spec/type_checking/function_call

```diff
@@ -80,35 +80,19 @@ component Main {
 --------------------------------------------------------------------------------
 module Maybe {
   fun nothing : Maybe(a) {
-    `new Nothing()`
+    ``
   }
 
   fun just (value : a) : Maybe(a) {
-    `new Just(value)`
+    ``
```

Comment:
why is it empty now?

Reply:
These tests does not execute JavaScript code, it just parses the code and type checks based on the annotations, so it does not make sense having that here.